app/src/main/java/org/p2p/wallet/auth/model/OnboardingFlow.kt

```diff
@@ -0,0 +1,13 @@
+package org.p2p.wallet.auth.model
+
+sealed interface OnboardingFlow {
+    object CreateWallet : OnboardingFlow
+
+    open class RestoreWallet : OnboardingFlow {
```

Comment:
sealed interfaces are working ok!

<img width="787" alt="Screen Shot 2022-09-15 at 11 28 36" src="https://user-images.githubusercontent.com/100791035/190354893-f4f08ecf-cb39-4240-9445-23a1c1c5089f.png">

Reply:
i used class, for set Restore without choosing any of flows, cause on first restore step i don't know current user flow